cmd/starter/main.go

```diff
@@ -76,98 +77,112 @@ func main() {
 	}, docker.AuthConfiguration{})
 	orFail(err)
 
-	redisClient.Incr("active-workers")
-	defer func() {
-		redisClient.Decr("active-workers")
-	}()
+	c := cron.New()
+	c.AddFunc("0 */5 * * * *", announceActiveWorker)
+	c.Start()
```

Comment:
It doesn't get removed currently… But as the set is capable of sorting and I query only the last 10 minutes the clients will expire itself. I'll add a set remove based on time to clean the redis database…